ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -540,6 +546,33 @@ public DataColumnSidecar deserializeSidecar(final Bytes serializedSidecar, final
         .sszDeserialize(serializedSidecar);
   }
 
+  public List<List<KZGProof>> deserializeDataColumnSidecarsProofs(final Bytes serializedProofs) {
```

Comment:
I think we normally have these DB-specific serializations\deserializations here: `tech.pegasys.teku.storage.server.kvstore.serialization`.

Is there a reason we can't push these down there?

Reply:
we also already have `toSSZBytes` `fromSSZBytes` serialization\deserialization functions in `KZGProof` itself. Maybe we could reuse them? (we will have a concatenation of SSZs instead of one SSZ with concatenated bytes, maybe it is equivalent)

Reply:
KzgProof's ssz functions looks junky, removed it.
I cannot remember what was my logic to avoid KVSerializer, tried it, looks ok: https://github.com/Consensys/teku/pull/10137/commits/7f1c51903ed8227fb4a05bcd8f13aec9bc17943d
So it's now single number only in serialization and all proofs.